VARIABLES.md

```diff
@@ -876,6 +916,14 @@ Default:  "{{ ssl_mutual_auth_enabled }}"
 
 ***
 
+### ksql_log_dir
+
+Only use to customize directory ksqlDB writes log files to. Default location is /var/log/confluent/ksql. NOTE- ksql.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that ksqlDB writes log files to. Default location is /var/log/confluent/ksql. NOTE- ksql.appender_log_path is deprecated.